airbyte-integrations/connectors/source-postgres-strict-encrypt/src/test-integration/java/io/airbyte/integrations/source/postgres_strict_encrypt/PostgresSourceStrictEncryptAcceptanceTest.java

```diff
@@ -2,7 +2,7 @@
  * Copyright (c) 2023 Airbyte, Inc., all rights reserved.
  */
 
-package io.airbyte.integrations.io.airbyte.integration_tests.sources;
+package io.airbyte.integrations.source.postgres_strict_encrypt;
```

Comment:
nit: Why the different package name structure?

Reply:
@akashkulk I followed the same structure that `Mysql` and `Mysql-strict-encrypt` have. The non strict-encrypt variant have their tests under `airbyte.integration_tests.sources`.

Mysql: [https://github.com/airbytehq/airbyte/blob/master/airbyte-integrations/connectors/source-mysql/src/test-integration/java/io/airbyte/integrations/io/airbyte/integration_tests/sources/MySqlSourceAcceptanceTest.java#L5](url)
Mysql-strict-encrypt: [https://github.com/airbytehq/airbyte/blob/master/airbyte-integrations/connectors/source-mysql-strict-encrypt/src/test-integration/java/io/airbyte/integrations/source/mysql_strict_encrypt/MySqlStrictEncryptSourceAcceptanceTest.java#L5](url)